### ISSUE TITLE
Have a way to validate accuracy with a test set of pro games

### ISSUE THREAD

**Issue by amj** [tensorflow/minigo#12]
A validation of value-error on pro games or kgs games would be nice to have

**Comment by amj** [comment]
@brilee we've got a script for this now, but i assume we want to automatically pull this into TB somehow?

**Comment by amj** [comment]
Ok, this is doable with `python main.py validate gs://path-to-pro-validation-dataset/ --load-file [recent model]` and can be run either by the main trainer or in parallel/backfill with some other job.

**Comment by brilee** [comment]
Can you add something to README with instructions on how to preprocess a pro validation set and filter it for komi=7.5 games?

What about a way to separate the namespace of the eval? On tensorboard, the pro eval set should appear as a separate curve from the selfplay eval set.

**Comment by amj** [comment]
Ok, i'll re-open.  I'll add instructions for preprocessing but i don't want to get into the grep/xargs/find weeds for posix/non-posix etc.

For separate namespace etc we can call it with a different logdir, no problem.

**Comment by amj** [comment]
Docs added in #104, closing.
